### DISCUSSION TITLE
Depth first / Breadth first rendering

### DISCUSSION THREAD

**Discussion by pointlesspun** [mermaid-js/mermaid#4178] [Q&A]
Hello all, 

Is there a way to force rendering of a flowchart to take a breadth first approach? Given a (largeish) specification, it seems the flowchart is always rendering it depth first regardless whether or not the nodes are declared depth first or breadth first.

Best,

**Comment by sidharthv96** [comment]
Rendering is done using dagre/elk. 
You can try using `flowchart-elk`
